src/labkey/security/Container.ts

```diff
@@ -85,6 +85,35 @@ export function deleteContainer(config: DeleteContainerOptions): XMLHttpRequest
     });
 }
 
+export interface RenameContainerOptions extends RequestCallbackOptions {
```

Comment:
So based on my reading of these options you could supply no parameters and receive a successful result?

Reply:
Either the name or the title is required, and the server-side code handles checking and validation. I thought that doing a more involved type for this situation seemed like overkill—but I have added a comment to make this more clear